rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -86,6 +89,8 @@ def __init__(self, parent=None):
         self.map_list_update_timer.timeout.connect(self.map_list_update)
         self.map_list_update_timer.start(1000)
 
+        self.save_map_image_dir =os.path.expanduser("~") + "/Desktop/map_images"
```

Comment:
missing space after `=`.

IMO we should have a common directory for the user files. I think @JuliusSustarevas was creating a directory ~/.rrviz for storing mission files.

Reply:
when i was writing this I asked @pepeRossRobotics  where he wanted them to be save and he said to put them on the users desktop in a folder where they could be easily found.

Reply:
Let's have a think how to approach this so that ideally we can decide on a single directory structure for all the user files. It will be way easier to tell user "If you want all your maps and paths on another machine then copy this directory" rather then to get X copy Y and to get A copy B. Let's make working with any files as easy for everyone as possible.

I assume the maps are only used for user to save it for whatever reason, we are not going to do anything with these maps, right?

Reply:
Ok sounds like a good plan. Yeah I pretty certain that's corrrect otherwise we'd want to save them on the robot rather than the ocu